servers/republik/modules/crowdfundings/graphql/resolvers/_mutations/claimMembership.js

```diff
@@ -5,23 +5,37 @@ const moment = require('moment')
 module.exports = async (_, args, {pgdb, req, t, mail: {enforceSubscriptions}}) => {
   ensureSignedIn(req)
 
-  // if this restriction gets removed, make sure to check if
-  // the membership doesn't already belong to the user, before
-  // making the the transfer and removing the voucherCode
-  if (await pgdb.public.memberships.count({userId: req.user.id})) {
+  // Throw an error if there is already an other active membership.
```

Comment:
Reading it now, I think everything from line 8 to 32 should also go into the transaction to the consistency of the state between querying and modifying.